tortoise.py

```diff
@@ -2,19 +2,36 @@
 
 
 def main():
-    
+    VALID_ARGS = ['tiny.tortoise', 'medium.tortoise', 'large.tortoise']
+
+    FILENAME = get_tortoise_file(sys.argv, VALID_ARGS)
+    if not '.tortoise' in FILENAME:
+        print(FILENAME)
+        exit()
+
+    content = get_file_content(FILENAME)
+    for i in content:
+        print(i, end='\n')
+
+
+def get_tortoise_file(argv, VALID_ARGS):
     try:
-        FILENAME = sys.argv[1]
+        FILENAME = argv[1]
     except IndexError:
-        print('\nYou have to specify what tortoise you want executed.\
-              \nPlease add it as an argument on the command line.\n')
-        return
+        return '\nPlease specify an argument on the command line\n'
+    if not argv[1] in VALID_ARGS:
```

Comment:
This isn't desired behavior. This would mean that the interpreter could only run Tortoise programs whose names are already hard-coded in this file, whereas we want to be able to run any Tortoise program. This mechanism needs to be removed.